packages/react/src/drafts/TabPanels/TabPanels.docs.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "tab_panels",
+  "name": "TabPanels",
+  "status": "draft",
+  "a11yReviewed": false,
+  "stories": [],
+  "props": [
+    {
+      "name": "aria-label",
+      "type": "string",
+      "description": "Used to set the `aria-label` on the `role=\"tablist\"` element."
+    },
+    {
+      "name": "children",
+      "type": "React.ReactNode",
+      "description": "The content of the component, can contain Tabs, Panels but also content before and after Tabs and after the Panels."
+    },
+    {
+      "name": "id",
+      "type": "string",
+      "description": "The id of the tab container, used to generate child ids."
+    },
+    {
+      "name": "selectedTabIndex",
+      "type": "number",
+      "description": "The 0-based index of the tab that is selected."
+    },
+    {
+      "name": "onTabContainerChange",
+      "type": "function",
+      "description": "Callback fired when the tab container changes (bubbles, cancelable): fired on `<tab-container>` before a new tab is selected and visibility is updated. `event.tab` is the tab that will be focused and `tab.panel` is the panel that will be shown if the event isn't cancelled."
+    },
+    {
+      "name": "onTabContainerChanged",
+      "type": "function",
+      "description": "Callback fired when the tab container changes (bubbles): fired on `<tab-container>` after a new tab is selected and visibility is updated. `event.tab` is the tab that is now active (and will be focused right after this event) and `event.panel` is the newly visible tab panel."
+    },
+    {
+      "name": "sx",
+      "type": "SystemStyleObject"
+    }
+  ],
+  "subcomponents": [
+    {
+      "name": "TabPanels.Tab",
```

Comment:
Should `Tab` have support for the `id` prop?

Reply:
It's available, but really I don't think we _want_ the user to set it. [See this comment for context](https://github.com/primer/react/pull/4241#discussion_r1517591820).

Reply:
I think `id` is a tricky one since generally we allow for folks to provide it and have a fallback if they don't. This pattern makes use of the `useId()` hook in the codebase (the one in `internal/hooks` versus the one directly from React) so when authoring it roughly looks like:

```tsx
function SomeComponent({ id }) {
  const someId = useId(id);
}
```

I think if it's a part of the public API then the component should follow this behavior but if it's not part of the public API then we wouldn't need to worry about it until someone requests it 👍